lib/tests/conftest.py

```diff
@@ -111,7 +111,7 @@ def pytest_configure(config: pytest.Config):
 
 
 def pytest_runtest_setup(item: pytest.Item):
-    is_require_snowflake = item.config.getoption("--require-snowflake")
+    is_require_snowflake = item.config.getoption("--require-snowflake", default=False)
```

Comment:
looks like this line from another PR, but anyway if merge conflict happen this will be very easy to fix :)

Reply:
I merged in develop, this should be resolved now